### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by camwes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,6 +109,15 @@ If everything is successful, you should see something like this:
 ```
 (master)$ Successfully rebased and updated refs/heads/master.
 ```
+### Possible issues with merging
+Safe merging strategy:
```
Comment (RichardLitt) [inline README.md]: Why is this safer?

Comment (camwes) [inline README.md]: from [git-scm](http://git-scm.com/docs/git-merge)
> With --no-commit perform the merge but pretend the merge failed and do not autocommit, to give the user a chance to inspect and further tweak the merge result before committing.

I think the additional opportunity to inspect code can be helpful. the ```no-ff``` flag doesn't really make anything "safer" it just maintains evidence that a feature branch once existed, which in the "branch for each task" style that a lot of teams I have worked, keeping the history consistent is important.

Comment (RichardLitt) [inline README.md]: Could you include this justification in the commit? It's good to know this.

Comment (camwes) [inline README.md]: for sure, I appreciate you being thorough

Comment (RichardLitt) [inline README.md]: Thanks for the thanks, not trying to block this, it's a great addition. Ping when you've forced pushed the changes and I'll merge them in.

**Comment by camwes** [comment]
@RichardLitt pushed explanation, we should be good

**Comment by RichardLitt** [comment]
Thanks!

**Pull request merged**